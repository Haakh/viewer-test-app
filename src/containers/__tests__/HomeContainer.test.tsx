import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { HomeContainer } from '..';

test('renders correctly', () => {
  const navigation: any = { navigate: jest.fn() };
  const tree = renderer.create(<HomeContainer navigation={navigation} />).toJSON();
  expect(tree).toMatchSnapshot();
});
