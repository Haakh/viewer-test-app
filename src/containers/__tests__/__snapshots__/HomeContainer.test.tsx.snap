// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  contentContainerStyle={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "grey",
        "flex": 1,
        "justifyContent": "center",
        "padding": 16,
      },
      Object {
        "backgroundColor": "#fff",
        "justifyContent": "flex-start",
        "padding": 0,
        "paddingTop": 22,
      },
    ]
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={true}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardSpace={0}
  onScroll={[Function]}
  resetKeyboardSpace={[Function]}
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollIntoView={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "grey",
    }
  }
  update={[Function]}
  viewIsInsideTabBar={false}
>
  <View>
    <View
      style={
        Object {
          "alignSelf": "stretch",
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-start",
            "marginLeft": 12,
          }
        }
      >
        <Text>
          ICO
        </Text>
        <Text>
           Lorem Ipsum - Dolor sit
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#555",
            "flex": 10,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
            }
          }
        >
          Hello
        </Text>
      </View>
      <View
        style={
          Object {
            "flex": 2.3,
          }
        }
      >
        <RCTScrollView
          contentContainerStyle={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            }
          }
          horizontal={true}
        >
          <View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#555",
                  "height": 150,
                  "justifyContent": "center",
                  "marginLeft": 5,
                  "opacity": 1,
                  "width": 150,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                  }
                }
              >
                Hi
              </Text>
            </View>
          </View>
        </RCTScrollView>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 3,
            "justifyContent": "center",
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#ddd",
              "borderColor": "#777",
              "borderRadius": 53.57142857142857,
              "borderWidth": 0.7,
              "height": 107.14285714285714,
              "justifyContent": "center",
              "opacity": 1,
              "width": 107.14285714285714,
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#fff",
                "borderColor": "#777",
                "borderRadius": 41.666666666666664,
                "borderWidth": 0.7,
                "height": 83.33333333333333,
                "width": 83.33333333333333,
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
